spec/features/idv/steps/gpo_step_spec.rb

```diff
@@ -103,12 +106,38 @@
         travel_to(days_passed.days.from_now) do
           sign_in_live_with_2fa(user)
           visit idv_request_letter_path
+
           expect(page).to have_current_path(idv_verify_by_mail_enter_code_path)
-          expect(page).not_to have_css('.usa-button', text: t('idv.buttons.mail.resend'))
+          expect(page).not_to have_link(
+            t('idv.gpo.did_not_receive_letter.intro.request_new_letter_link'),
+          )
         end
       end
     end
 
+    context 'too little time has passed', :js do
```

Comment:
How is 'too little time' being set up here?

Reply:
That's the default state; right after you request a letter, it's too soon to request another.

Reply:
Can `complete_idv_and_sign_out` be renamed `complete_idv_by_mail_and_sign_out`? It took me quite a while to figure out that that's where the first letter is being requested.

Reply:
Renamed.